src/components/App/App.js

```diff
@@ -0,0 +1,31 @@
+import React, { useState, useEffect } from 'react'
+import { Routes, Route } from 'react-router-dom'
+import './App.css';
+import Recommended from '../Recommended/Recommended'
+import { userData } from './userData'
+import { getTrendingAlbums, getRecommendedAlbums } from '../../api'
+
+const App = () => {
+  const [recommendedData, setRecommendedData] = useState()
+  const [user, setUser] = useState(userData[0])
+
+  useEffect(() => {
+    if (!user.collection.length) {
+      getTrendingAlbums(setRecommendedData)
+    } else {
+      const albumID = user.collection[Math.floor(Math.random() * user.collection.length)]
```

Comment:
There may have been some misunderstanding. This should happen only for the Discover tab no? The homepage should always only show trending.